python3Packages.{langchain,langgraph}: 250730 update

sarahec opened the pull request:
1. Ran update script
2. Replaced `pdm-backend` with `hatchling` as needed to keep up with upstream
3. Updated prerequisites `python3Packages.anthropic` and `python3Packages.orgmsgpack` to match requirements

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [x] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

sarahec committed:
python312Packages.langgraph: 0.4.1 -> 0.6.1

https://github.com/langchain-ai/langgraph/releases/tag/0.6.1

sarahec committed:
python312Packages.langchain-ollama: 0.3.4 -> 0.3.6

changelog: https://github.com/langchain-ai/langchain/releases/tag/langchain-ollama%3D%3D0.3.6
diff: https://github.com/langchain-ai/langchain/compare/langchain-ollama%3D%3D0.3.4...langchain-ollama%3D%3D0.3.6

sarahec committed:
python312Packages.langgraph-checkpoint: 2.0.26 -> 2.1.1

https://github.com/langchain-ai/langgraph/releases/tag/checkpoint%3D%3D2.1.0
https://github.com/langchain-ai/langgraph/releases/tag/checkpoint%3D%3D2.1.1

sarahec committed:
python312Packages.langgraph-sdk: 0.1.69 -> 0.2.0

https://github.com/langchain-ai/langgraph/releases/tag/sdk%3D%3D0.2.0

sarahec committed:
python312Packages.langgraph-checkpoint-sqlite: 2.0.6 -> 2.0.10

https://github.com/langchain-ai/langgraph/releases/tag/checkpointsqlite%3D%3D2.0.7
https://github.com/langchain-ai/langgraph/releases/tag/checkpointsqlite%3D%3D2.0.8
https://github.com/langchain-ai/langgraph/releases/tag/checkpointsqlite%3D%3D2.0.9
https://github.com/langchain-ai/langgraph/releases/tag/checkpointsqlite%3D%3D2.0.10

sarahec committed:
python312Packages.langchain-core: 0.3.66 -> 0.3.72

https://github.com/langchain-ai/langchain/releases/tag/langchain-core%3D%3D0.3.72

sarahec committed:
python312Packages.langchain-anthropic: 0.3.15 -> 0.3.17

https://github.com/langchain-ai/langchain/releases/tag/langchain-anthropic%3D%3D0.3.16
https://github.com/langchain-ai/langchain/releases/tag/langchain-anthropic%3D%3D0.3.17

sarahec committed:
python312Packages.langgraph-checkpoint-postgres: 2.0.21 -> 2.0.23

https://github.com/langchain-ai/langgraph/releases/tag/checkpointpostgres%3D%3D2.0.22
https://github.com/langchain-ai/langgraph/releases/tag/checkpointpostgres%3D%3D2.0.23

sarahec committed:
python312Packages.langgraph-prebuilt: 0.1.8 -> 0.6.1

https://github.com/langchain-ai/langgraph/releases/tag/prebuilt%3D%3D0.6.1

sarahec committed:
python3Packages.ormsgpack: 1.9.1 -> 1.10.0

changelog: https://github.com/aviramha/ormsgpack/releases/tag/1.10.0
diff: https://github.com/aviramha/ormsgpack/compare/1.9.1...1.10.0

sarahec committed:
python3Packages.langchain: disable tests that break under new -core

langchain-core has gotten ahead of langchain, and deprecated code is
causing langchain's tests to break. Disabled with a note to fix on
update.

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 427212`
Commit: `06f5900f9a22dcbc59a0c7625e3d090fb852257c`

---
### `aarch64-darwin`
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 193 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.ormsgpack</li>
    <li>python312Packages.ormsgpack.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.anthropic</li>
    <li>python313Packages.anthropic.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-sdk</li>
    <li>python313Packages.langgraph-sdk.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.ormsgpack</li>
    <li>python313Packages.ormsgpack.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.langchain</summary>
<pre>stripping (with command strip and flags -S) in  /nix/store/3ph1z3pk7g0vzggl1xicbcpc3xx8hjgi-python3.12-langchain-0.3.26/lib
checking for references to /private/tmp/nix-build-python3.12-langchain-0.3.26.drv-0/ in /nix/store/8x8sb2ppc0vn78nnhhl8zlmza87vl1dv-python3.12-langchain-0.3.26-dist...
patching script interpreter paths in /nix/store/8x8sb2ppc0vn78nnhhl8zlmza87vl1dv-python3.12-langchain-0.3.26-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: langchain
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
Disabled tests path glob "tests/unit_tests/indexes/test_hashed_indexing.py" does not match any paths. Aborting</pre>
</details>
<details>
<summary>python312Packages.langchain-xai</summary>
<pre>1 snapshot failed.

----------------------------------------------- benchmark: 1 tests -----------------------------------------------
Name (time in ms)          Min       Max      Mean  StdDev    Median     IQR  Outliers     OPS  Rounds  Iterations
------------------------------------------------------------------------------------------------------------------
test_init_time        375.8723  383.4294  379.3968  3.4860  377.8473  6.2603       2;0  2.6358       5           1
------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
============================= slowest 5 durations ==============================
3.66s call     tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_init_time
1.37s call     tests/unit_tests/test_chat_models.py::test_initialization
1.25s call     tests/unit_tests/test_chat_models.py::test_xai_model_param
1.21s call     tests/unit_tests/test_chat_models.py::test_chat_xai_extra_kwargs
0.69s setup    tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_bind_tool_pydantic
=========================== short test summary info ============================
FAILED tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_serdes - AssertionError: assert [+ received] == [- snapshot]
================== 1 failed, 23 passed, 2 warnings in 18.09s ===================</pre>
</details>
<details>
<summary>python313Packages.langchain</summary>
<pre>stripping (with command strip and flags -S) in  /nix/store/1cxmc8y345615zi5d3i4fdkz8zsj2c46-python3.13-langchain-0.3.26/lib
checking for references to /private/tmp/nix-build-python3.13-langchain-0.3.26.drv-0/ in /nix/store/lkqm783712l1z6bh2ib8fpk5qci2pffw-python3.13-langchain-0.3.26-dist...
patching script interpreter paths in /nix/store/lkqm783712l1z6bh2ib8fpk5qci2pffw-python3.13-langchain-0.3.26-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: langchain
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
Disabled tests path glob "tests/unit_tests/indexes/test_hashed_indexing.py" does not match any paths. Aborting</pre>
</details>
<details>
<summary>python313Packages.langchain-xai</summary>
<pre>1 snapshot failed.

------------------------------------------------ benchmark: 1 tests ------------------------------------------------
Name (time in ms)          Min       Max      Mean   StdDev    Median      IQR  Outliers     OPS  Rounds  Iterations
--------------------------------------------------------------------------------------------------------------------
test_init_time        360.8505  388.6530  374.7051  11.3457  370.9324  17.9428       2;0  2.6688       5           1
--------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
============================= slowest 5 durations ==============================
3.62s call     tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_init_time
2.13s call     tests/unit_tests/test_chat_models.py::test_initialization
1.34s call     tests/unit_tests/test_chat_models.py::test_xai_model_param
1.12s call     tests/unit_tests/test_chat_models.py::test_chat_xai_extra_kwargs
0.66s call     tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_init
=========================== short test summary info ============================
FAILED tests/unit_tests/test_chat_models_standard.py::TestXAIStandard::test_serdes - AssertionError: assert [+ received] == [- snapshot]
================== 1 failed, 23 passed, 2 warnings in 18.02s ===================</pre>
</details>
</details>

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 427212`
Commit: `955457692460a8466711407f772d0d597af2e1b7`

---
### `x86_64-linux`
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 208 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat.dist</li>
    <li>gnome-frog</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>home-assistant-component-tests.anthropic</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>newelle</li>
    <li>open-interpreter (python313Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python313Packages.open-interpreter.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mmengine</li>
    <li>python312Packages.mmengine.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.ormsgpack</li>
    <li>python312Packages.ormsgpack.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.anthropic</li>
    <li>python313Packages.anthropic.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-sdk</li>
    <li>python313Packages.langgraph-sdk.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.ormsgpack</li>
    <li>python313Packages.ormsgpack.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.langgraph-prebuilt</summary>
<pre>/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/__init__.py:71: in <module>
    from langgraph.pregel.algo import (
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/algo.py:73: in <module>
    from langgraph.pregel.read import INPUT_CACHE_KEY_TYPE, PregelNode
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/read.py:14: in <module>
    from langgraph.pregel.protocol import PregelProtocol
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/protocol.py:16: in <module>
    class PregelProtocol(Runnable[InputT, Any], Generic[StateT, InputT, OutputT], ABC):
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases ABC, Generic
stopping postgresql
waiting for server to shut down....2025-07-22 17:05:12.392 UTC [217] LOG:  received fast shutdown request
2025-07-22 17:05:12.397 UTC [217] LOG:  aborting any active transactions
2025-07-22 17:05:12.399 UTC [217] LOG:  background worker "logical replication launcher" (PID 223) exited with exit code 1
2025-07-22 17:05:12.399 UTC [218] LOG:  shutting down
2025-07-22 17:05:12.407 UTC [218] LOG:  checkpoint starting: shutdown immediate
2025-07-22 17:05:12.754 UTC [218] LOG:  checkpoint complete: wrote 934 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.041 s, sync=0.278 s, total=0.356 s; sync files=308, longest=0.006 s, average=0.001 s; distance=4259 kB, estimate=4259 kB; lsn=0/1915748, redo lsn=0/1915748
2025-07-22 17:05:12.762 UTC [217] LOG:  database system is shut down
 done
server stopped</pre>
</details>
<details>
<summary>python313Packages.langgraph-prebuilt</summary>
<pre>    from langgraph.pregel.checkpoint import copy_checkpoint
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/__init__.py:71: in <module>
    from langgraph.pregel.algo import (
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/algo.py:73: in <module>
    from langgraph.pregel.read import INPUT_CACHE_KEY_TYPE, PregelNode
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/read.py:14: in <module>
    from langgraph.pregel.protocol import PregelProtocol
/nix/store/sphvvrxc9ia2c0scrihy4fjbnh935zjr-source/libs/langgraph/langgraph/pregel/protocol.py:16: in <module>
    class PregelProtocol(Runnable[InputT, Any], Generic[StateT, InputT, OutputT], ABC):
E   TypeError: Cannot create a consistent method resolution order (MRO) for bases ABC, Generic
stopping postgresql
waiting for server to shut down....2025-07-22 17:05:12.355 UTC [217] LOG:  received fast shutdown request
2025-07-22 17:05:12.365 UTC [217] LOG:  aborting any active transactions
2025-07-22 17:05:12.367 UTC [217] LOG:  background worker "logical replication launcher" (PID 223) exited with exit code 1
2025-07-22 17:05:12.367 UTC [218] LOG:  shutting down
2025-07-22 17:05:12.372 UTC [218] LOG:  checkpoint starting: shutdown immediate
2025-07-22 17:05:12.740 UTC [218] LOG:  checkpoint complete: wrote 934 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.039 s, sync=0.300 s, total=0.374 s; sync files=308, longest=0.009 s, average=0.001 s; distance=4259 kB, estimate=4259 kB; lsn=0/1915748, redo lsn=0/1915748
2025-07-22 17:05:12.748 UTC [217] LOG:  database system is shut down
 done
server stopped</pre>
</details>
</details>

eu90h commented on pkgs/development/python-modules/langchain-google-genai/default.nix:
Looks good on this package. I verified the hash number with `nix-prefetch-github`. I then ran a test with `nixpkgs-review pr -p python312Packages.langchain-google-genai --checkout commit 427212` and everything appears to be in working order.

sarahec committed:
python3Packages.langchain: 0.3.26 -> 0.3.27

https://github.com/langchain-ai/langchain/releases/tag/langchain%3D%3D0.3.27
https://github.com/langchain-ai/langchain/compare/langchain%3D%3D0.3.26...langchain%3D%3D0.3.27

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 427212`
Commit: `1cea994d2be7fb016b77d13546dd0337eaf99ab2`

---
### `aarch64-darwin`
<details>
  <summary>:x: 34 packages failed to build:</summary>
  <ul>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 104 packages built:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.ormsgpack</li>
    <li>python312Packages.ormsgpack.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-sdk</li>
    <li>python313Packages.langgraph-sdk.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.ormsgpack</li>
    <li>python313Packages.ormsgpack.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.langchain-anthropic</summary>
<pre> - Adding langchain_anthropic/__init__.py -> langchain_anthropic/__init__.py
 - Adding langchain_anthropic/_client_utils.py -> langchain_anthropic/_client_utils.py
 - Adding langchain_anthropic/chat_models.py -> langchain_anthropic/chat_models.py
 - Adding langchain_anthropic/experimental.py -> langchain_anthropic/experimental.py
 - Adding langchain_anthropic/llms.py -> langchain_anthropic/llms.py
 - Adding langchain_anthropic/output_parsers.py -> langchain_anthropic/output_parsers.py
 - Adding langchain_anthropic/py.typed -> langchain_anthropic/py.typed
 - Adding langchain_anthropic-0.3.17.dist-info/RECORD -> langchain_anthropic-0.3.17.dist-info/RECORD
Successfully built langchain_anthropic-0.3.17-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic/dist /private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Unpacking to: unpacked/langchain_anthropic-0.3.17...OK
Repacking wheel as ./langchain_anthropic-0.3.17-py3-none-any.whl...OK
/private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for langchain_anthropic-0.3.17-py3-none-any.whl
  - anthropic<1,>=0.57.0 not satisfied by version 0.55.0</pre>
</details>
<details>
<summary>python312Packages.langchain-openai</summary>
<pre>benchmark: 5.1.0 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /private/tmp/nix-build-python3.12-langchain-openai-0.3.28.drv-0/source/libs/partners/openai
configfile: pyproject.toml
plugins: anyio-4.9.0, langsmith-0.3.45, asyncio-0.26.0, benchmark-5.1.0, codspeed-3.2.0, recording-0.13.4, socket-0.7.0, syrupy-4.9.1, cov-nixpkgs-stub-1.0.0, mock-3.14.1, requests-mock-1.12.1
asyncio: mode=Mode.AUTO, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... 
collected 237 items / 1 error / 37 deselected / 200 selected                   

==================================== ERRORS ====================================
____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
tests/unit_tests/chat_models/test_responses_stream.py:234: in <module>
    ResponseTextDeltaEvent(
E   pydantic_core._pydantic_core.ValidationError: 1 validation error for ResponseTextDeltaEvent
E   logprobs
E     Field required [type=missing, input_value={'content_index': 0, 'del...onse.output_text.delta'}, input_type=dict]
E       For further information visit https://errors.pydantic.dev/2.11/v/missing
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 37 deselected, 1 error in 0.56s ========================</pre>
</details>
<details>
<summary>python313Packages.langchain-anthropic</summary>
<pre> - Adding langchain_anthropic/__init__.py -> langchain_anthropic/__init__.py
 - Adding langchain_anthropic/_client_utils.py -> langchain_anthropic/_client_utils.py
 - Adding langchain_anthropic/chat_models.py -> langchain_anthropic/chat_models.py
 - Adding langchain_anthropic/experimental.py -> langchain_anthropic/experimental.py
 - Adding langchain_anthropic/llms.py -> langchain_anthropic/llms.py
 - Adding langchain_anthropic/output_parsers.py -> langchain_anthropic/output_parsers.py
 - Adding langchain_anthropic/py.typed -> langchain_anthropic/py.typed
 - Adding langchain_anthropic-0.3.17.dist-info/RECORD -> langchain_anthropic-0.3.17.dist-info/RECORD
Successfully built langchain_anthropic-0.3.17-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic/dist /private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Unpacking to: unpacked/langchain_anthropic-0.3.17...OK
Repacking wheel as ./langchain_anthropic-0.3.17-py3-none-any.whl...OK
/private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for langchain_anthropic-0.3.17-py3-none-any.whl
  - anthropic<1,>=0.57.0 not satisfied by version 0.55.0</pre>
</details>
<details>
<summary>python313Packages.langchain-openai</summary>
<pre>configfile: pyproject.toml
plugins: anyio-4.9.0, langsmith-0.3.45, asyncio-0.26.0, benchmark-5.1.0, codspeed-3.2.0, recording-0.13.4, socket-0.7.0, syrupy-4.9.1, cov-nixpkgs-stub-1.0.0, mock-3.14.1, requests-mock-1.12.1
asyncio: mode=Mode.AUTO, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... 
collecting 5 items                                                             
collecting 177 items / 1 error                                                 
collected 237 items / 1 error / 37 deselected / 200 selected                   

==================================== ERRORS ====================================
____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
tests/unit_tests/chat_models/test_responses_stream.py:234: in <module>
    ResponseTextDeltaEvent(
E   pydantic_core._pydantic_core.ValidationError: 1 validation error for ResponseTextDeltaEvent
E   logprobs
E     Field required [type=missing, input_value={'content_index': 0, 'del...onse.output_text.delta'}, input_type=dict]
E       For further information visit https://errors.pydantic.dev/2.11/v/missing
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 37 deselected, 1 error in 1.33s ========================</pre>
</details>
</details>
